include/ck_tile/ops/fmha/pipeline/block_fmha_fwd_splitkv_pipeline_qr_ks_vs.hpp

```diff
@@ -271,14 +278,14 @@ struct BlockFmhaFwdSplitKVPipelineQRKSVS
         {
             // STAGE 1, QK gemm
             auto k_dram_window = make_tile_window(
-                k_dram_block_window.get_bottom_tensor_view(),
-                k_dram_block_window.get_window_lengths(),
-                k_dram_block_window.get_window_origin(),
+                k_dram_block_window,
                 Policy::template MakeKDramTileDistribution<Problem>()); // K DRAM tile window for
                                                                         // load
 
             auto k_block_tile = load_tile(k_dram_window);
             {
+                // moving k_dram_window is an in-page-block operation, so there is
+                // no need to invoke k_page_block_navigator.move_tile_window() here.
                 move_tile_window(k_dram_window, {0, kK0});
```

Comment:
Does this `k_dram_window` only move within the current page block?

Reply:
I created a type `PageBlockNavigator` to handle paged k/v cache related operations. If it is possible to move tile window across pages, we will have to invoke `PageBlockNavigator::move_tile_window()` to do necessary checks and update underlying descriptor & pointer. Otherwise, we don't have to do such things. the `move_tile_window()` here is moving tile on hdim_q dimension, hence no check needed.

Reply:
Yes, `k_dram_window` only move within 1 page block, move on 2nd dimention of (page_block_size, hdim_q) memory.

Reply:
Thanks!